ignite/distributed/utils.py

```diff
@@ -321,7 +324,28 @@ def train_fn(local_rank, a, b, c, d=12):
         )
 
 
-def all_reduce(tensor: Union[torch.Tensor, float], op: str = "SUM") -> Union[torch.Tensor, float]:
+def new_group(group: Any) -> Any:
+
+    if group is None:
+        return None
+    elif isinstance(group, list) and all(isinstance(item, int) for item in group):
+        group = [group]
+    elif all(isinstance(item, int) for list_ in group for item in list_):
+        group = group
+    else:
+        raise ValueError("Group should be list or list of list")
+
+    if idist.backend() in ["nccl", "gloo", "mpi"]:
+        return dist.new_group(ranks=group[0])
+    elif idist.backend() in ["xla-tpu"]:
+        return group
+    elif idist.backend() == "horovod":
+        from horovod.common.process_sets import ProcessSet
```

Comment:
I think it is better to call `_model.new_group` instead of doing a dispatch here